src/searching.md

```diff
@@ -95,86 +94,86 @@ Things to note from the above:
 ## Limiting to a field
 
 You can also ask Anki to match only if a particular field contains some
-text. Unlike the searches above, searching on fields requires an 'exact
-match' by default.
+text. Unlike the previous search examples, searching in fields requires an exact
+match by default.
 
 `front:dog`\
-find notes with a Front field of exactly "dog". A field that says "a
+finds notes with a Front field of exactly "dog". A field that says "a
 dog" will not match.
 
 `"animal front:a dog"`\
-find notes where the "Animal Front" field is exactly "a dog". The double quotes are
-mandatory, see [below](#matching-special-characters).
+finds notes where the "Animal Front" field is exactly "a dog". The double quotes are
+mandatory, see [this section](#matching-special-characters) of the page.
 
 `front:*dog*`\
-find notes with Front field containing dog somewhere
+finds notes where the Front field contains dog somewhere.
 
 `front:`\
-find notes that have an empty Front field
+finds notes that have an empty Front field.
 
 `front:_*`\
-find notes that have a non-empty Front field
+findd notes that have a non-empty Front field.
 
 `front:*`\
-find notes that have a Front field, empty or not
+finds notes that have a Front field, empty or not.
 
 `fr*:text`\
-find notes in a field starting with "fr". Requires Anki 2.1.24+ or AnkiMobile 2.1.60+.
+finds notes in a field starting with "fr". Requires Anki 2.1.24+, AnkiMobile 2.1.60+, or AnkiDroid 2.17+.
 
 ## Tags, decks, cards and notes
 
 `tag:animal`\
-find notes with the tag "animal", or subtags like "animal::mammal"
+finds notes with the tag "animal", or subtags like "animal::mammal".
 
 `tag:none`\
-find notes with no tags
+finds notes with no tags.
 
 `tag:ani*`\
-find notes with tags starting with ani
+finds notes with tags starting with ani.
 
 `deck:french`\
-find cards in a French deck, or subdecks like French::Vocab
+find cards in a deck called "French", or its subdecks like French::Vocab.
```

Comment:
Should add quotation marks to subdeck name as well, to match the other examples:
`or its subdecks like "French::Vocab".`

Reply:
```suggestion
find cards in a deck called "French", or its subdecks like "French::Vocab".
```